src/nvim/api/command.c

```diff
@@ -1015,11 +1015,11 @@ void create_user_command(uint64_t channel_id, String name, Object command, Dict(
   uint32_t argt = 0;
   int64_t def = -1;
   cmd_addr_T addr_type_arg = ADDR_NONE;
-  int compl = EXPAND_NOTHING;
-  char *compl_arg = NULL;
+  int comp = EXPAND_NOTHING;
+  char *comp_arg = NULL;
   const char *rep = NULL;
   LuaRef luaref = LUA_NOREF;
-  LuaRef compl_luaref = LUA_NOREF;
+  LuaRef comp_luaref = LUA_NOREF;
```

Comment:
Is anything with a `compl_` prefix reserved?

Reply:
No, just `compl` as an alternative syntax for the bitwise `~`, but I renamed other variables too just to be a little bit more consistent.